sql/mob_spawn_points.sql

```diff
@@ -49569,8 +49569,8 @@ INSERT INTO `mob_spawn_points` VALUES (17457309,'Tros','Tros',28,-289,-45,212,23
 INSERT INTO `mob_spawn_points` VALUES (17461249,'Bloodsucker','Bloodsucker',1,-140,16,-231,127);
 INSERT INTO `mob_spawn_points` VALUES (17461250,'Acid_Grease','Acid Grease',2,0,0,0,0);
 INSERT INTO `mob_spawn_points` VALUES (17461251,'Bloodsucker','Bloodsucker',1,-140,16,-216,127);
-INSERT INTO `mob_spawn_points` VALUES (17461252,'Mousse','Mousse',3,0,0,0,0);
-INSERT INTO `mob_spawn_points` VALUES (17461253,'Mousse','Mousse',3,0,0,0,0);
+-- INSERT INTO `mob_spawn_points` VALUES (17461252,'Mousse','Mousse',3,0,0,0,0);
+-- INSERT INTO `mob_spawn_points` VALUES (17461253,'Mousse','Mousse',3,0,0,0,0);
```

Comment:
this isn't needed. we normally do not comment out rows. Mobs w3ith a position of 0,0,0 normally will not spawn. To fix them, the correct position must be given.

Reply:
```sql

INSERT INTO `mob_spawn_points` VALUES (17461300,'Funnel_Bats','Funnel Bats',4,-24.162,0.012,-102.723,29);
INSERT INTO `mob_spawn_points` VALUES (17461301,'Mousse','Mousse',3,-21.231,0.968,-123.463,187);
INSERT INTO `mob_spawn_points` VALUES (17461302,'Mousse','Mousse',3,-32.273,1.000,-99.785,150);
INSERT INTO `mob_spawn_points` VALUES (17461303,'Mousse','Mousse',3,-19.582,0.954,-125.114,31);
INSERT INTO `mob_spawn_points` VALUES (17461304,'Mousse','Mousse',3,-19.178,0.964,-147.278,7);
INSERT INTO `mob_spawn_points` VALUES (17461305,'Mousse','Mousse',3,-20.786,1.047,-147.246,137);
INSERT INTO `mob_spawn_points` VALUES (17461306,'Mousse','Mousse',3,-20.579,0.998,-160.756,172);
INSERT INTO `mob_spawn_points` VALUES (17461307,'Sewer_Syrup','Sewer Syrup',10,-20.321,0.500,-160.689,59);
```
id 17461306 is the retail placeholder, currently using group id 3

Reply:
idk why these mobs were changed to group 3 and group 15 was changed/deleted. Group 3 are the Mousse that are fished up, and are supposed to have a scripted spawn. Group 15 was being used for roaming Mousse, but looks like it was adjusted instead for the higher level Panna Cotta.

Reply:
Ok I removed the unnecessary comments.

Other than that, it's not very clear what I should do...
What is sure is that Mousse were missing, and with my PR they are back, and I tested that the NM can be poped by killing them.

Reply:
Mousse needs a new group for IDs 17461301-17461306. IDs 17461252 and 17461253 are fished up and have different stats/drops/spawntype. [See the first two entries on this page.](https://ffxiclopedia.fandom.com/wiki/Mousse)